scripts/config/msu/text.nut

```diff
@@ -1,7 +1,7 @@
 ::MSU.Text <- {
 	function color( _color, _string )
 	{
-		return format("[color=%s]%s[/color]", _color, _string);
+		return "[color=" + _color + "]" + _string + "[/color]";
 	}
 
 	function colorGreen( _string )
```

Comment:
looks like bad spacing here or is that just github

Reply:
Below the lines indicated here

Reply:
It looks fine to me?

Reply:
then it is probably github